ramls/instance.json

```diff
@@ -400,6 +400,108 @@
       }
     }
   },
+  "precedingTitles": {
+    "description": "Array of preceding titles",
+    "type": "array",
+    "items": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Id of the preceding title",
+          "type": "string",
+          "$ref": "uuid.json"
```

Comment:
replace please all references to uuid.json with this "raml-storage/schemas/common/uuid.json"